test/grpc/route_guide/route_guide.proto

```diff
@@ -0,0 +1,108 @@
+// Copyright 2015 gRPC authors.
```

Comment:
Rather than copying this file can we just reference it from `@com_github_grpc_grpc` similar to what we do with `helloworld.proto`?